python/libs/BIDS.py

```diff
@@ -15,6 +15,12 @@ def __init__(self, bids_directory):
             for filename in files:
                 if filename == '.bidsignore':
                     continue
+
+                if filename.endswith('_annotations.tsv'):
```

Comment:
I think we will stop supporting those files

Reply:
so we will be supporting _annotations.json for recording parameters input and _events.tsc for events input only correct? If so, this condition can be removed